src/contracts/core/DelegationManager.sol

```diff
@@ -97,7 +102,14 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
         unchecked {
             nonces[staker] = nonce + 1;
         }
-        bytes32 digestHash = keccak256(abi.encodePacked("\x19\x01", DOMAIN_SEPARATOR, structHash));
+
+        bytes32 digestHash;
+        if (block.chainid != ORIGINAL_CHAIN_ID) {
+            bytes32 domain_separator = keccak256(abi.encode(DOMAIN_TYPEHASH, bytes("EigenLayer"), block.chainid, address(this)));
```

Comment:
4.5.B is again relevant here

Reply:
Addressed: [3d1b89c](https://github.com/Layr-Labs/eigenlayer-contracts/pull/14/commits/3d1b89c1c2e0627bf2e5679d6e7588119f23a829)